app/assets/stylesheets/layout/common/_header.scss

```diff
@@ -1,7 +1,7 @@
 .l-header {
-  background: $color-green-primary;
```

Comment:
Might be worth adding this to the header element CSS instead if the layout container within it